src/current/_includes/releases/v24.2/v24.2.0-beta.2.md

```diff
@@ -0,0 +1,50 @@
+## v24.2.0-beta.2
+
+Release Date: July 24, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-0-beta-2-security-updates">Security updates</h3>
+
+- URLs in the following SQL statements are now sanitized of any secrets before being written to [unredacted logs]({% link v24.2/configure-logs.md %}#redact-logs). [#127360][#127360]
+- [`ALTER BACKUP SCHEDULE`]({% link v24.2/alter-backup.md %})
+- [`ALTER BACKUP`]({% link v24.2/alter-backup.md %})
+- [`ALTER CHANGEFEED SET sink`]({% link v24.2/alter-changefeed.md %})
+- [`BACKUP`]({% link v24.2/backup.md %})
+- [`COPY`]({% link v24.2/copy-from.md %})
+- [`CREATE CHANGEFEED`]({% link v24.2/create-changefeed.md %})
+- [`CREATE EXTERNAL CONNECTION`]({% link v24.2/create-external-connection.md %})
+- [`CREATE SCHEDULE FOR BACKUP`]({% link v24.2/create-schedule-for-backup.md %})
+- [`CREATE SCHEDULE FOR CHANGEFEED`]({% link v24.2/create-schedule-for-changefeed.md %})
+- [`EXPORT`]({% link v24.2/export.md %})
+- [`IMPORT`]({% link v24.2/import.md %})
```

Comment:
```suggestion
- [`IMPORT`]({% link v24.2/import-into.md %})
```
Should `IMPORT INTO` also be in this list? Should the link text here be updated?

Reply:
based on looking at [this PR](https://github.com/cockroachdb/cockroach/pull/127360/commits/ee740dde8c73b4c07a0d3e4333e3cf032f6d9678) it changes both `IMPORT` and `IMPORT INTO`, so I'll add `IMPORT INTO` to the list